diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index eebdf42eb721b756d4f554b09b943774faa659b0..0000000000000000000000000000000000000000
diff --git a/lib/engines.js b/lib/engines.js
index 38f993db06cf364191ac635a6590c2d29303297d..24f6f0b48a2a4e0e394830f629ed8625587859f4 100644
--- a/lib/engines.js
+++ b/lib/engines.js
@@ -1,20 +1,20 @@
-'use strict';
+"use strict";
 
-const yaml = require('js-yaml');
+const yaml = require("js-yaml");
 
 /**
  * Default engines
  */
 
-const engines = exports = module.exports;
+const engines = (exports = module.exports);
 
 /**
  * YAML
  */
 
 engines.yaml = {
-  parse: yaml.safeLoad.bind(yaml),
-  stringify: yaml.safeDump.bind(yaml)
+  parse: yaml.load.bind(yaml),
+  stringify: yaml.dump.bind(yaml),
 };
 
 /**
@@ -23,10 +23,10 @@ engines.yaml = {
 
 engines.json = {
   parse: JSON.parse.bind(JSON),
-  stringify: function(obj, options) {
-    const opts = Object.assign({replacer: null, space: 2}, options);
+  stringify: function (obj, options) {
+    const opts = Object.assign({ replacer: null, space: 2 }, options);
     return JSON.stringify(obj, opts.replacer, opts.space);
-  }
+  },
 };
 
 /**
@@ -38,7 +38,7 @@ engines.javascript = {
     /* eslint no-eval: 0 */
     try {
       if (wrap !== false) {
-        str = '(function() {\nreturn ' + str.trim() + ';\n}());';
+        str = "(function() {\nreturn " + str.trim() + ";\n}());";
       }
       return eval(str) || {};
     } catch (err) {
@@ -48,7 +48,7 @@ engines.javascript = {
       throw new SyntaxError(err);
     }
   },
-  stringify: function() {
-    throw new Error('stringifying JavaScript is not supported');
-  }
+  stringify: function () {
+    throw new Error("stringifying JavaScript is not supported");
+  },
 };
